tokio/src/runtime/handle.rs

```diff
@@ -301,7 +301,7 @@ impl Handle {
             .expect("failed to park thread")
     }
 
-    pub(crate) fn shutdown(mut self) {
+    pub(crate) fn shutdown(&mut self) {
         self.spawner.shutdown();
     }
```

Comment:
Why this?

Reply:
Adding a destructor to `Runtime` means we can no longer move out of the `handle` field, which would have made it impossible to call `shutdown()` on it. This was the simplest fix.

Reply:
(See the implementation of `shutdown_timeout`)